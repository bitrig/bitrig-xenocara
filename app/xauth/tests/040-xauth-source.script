#!/bin/bash

# setup the test ennvironment inside this shell
. setup-sourced

# Execute the test
CMDFILE=$DATADIR/xauth-cmdfile
echo add examplehost1/unix:0 . b90b0fd1cf6a0e7a2c74c00000000001 > $CMDFILE
xauth_silent source $CMDFILE
xauth list
