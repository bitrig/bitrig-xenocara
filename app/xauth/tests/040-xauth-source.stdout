> xauth some silent commands which should not be logged...
  exits with 0
> xauth list
examplehost1/unix:0  MIT-MAGIC-COOKIE-1  b90b0fd1cf6a0e7a2c74c00000000001
  exits with 0
